concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1370,34 +1370,25 @@ jobs:
                 < cf-tfstate/cf.tfstate \
                 > cf-terraform-outputs.yml
 
-                if [ -s cf-vars-store/cf-vars-store.yml ]
-                then
-                  echo cf-vars-store.yml has vars to copy.
-                else
-                  echo cf-vars-store.yml is empty. Skipping ...
-                  exit 0
-                fi
-
+                echo 'Logging into Credhub'
+                credhub api -s "$BOSH_ENVIRONMENT:8844/api"
+                credhub login --client-name="${CREDHUB_CLIENT_ID}" --client-secret="${CREDHUB_CLIENT_SECRET}"
```

Comment:
Since we already have the environment variables, can we do implicit auth?

> credhub login --help
> Usage:
>   credhub [OPTIONS] login [login-OPTIONS]
> 
> Authenticate with CredHub. UAA password and client credential grants are supported. **If client credentials exist in the environment, authentication will be performed automatically without the need to explicitly call this command**.